It is detecting GNU hurd properly (see https://buildd.debian.org/status/fetch.php?pkg=j4-dmenu-desktop&arch=hurd-amd64&ver=3.2-3&stamp=1777427113&raw=0).

But I notice just now a warning that may be only in Debian's builds due to the build flags used:

```
In file included from src/j4-dmenu-desktop.p/j4-dmenu-desktop-unity0.cpp:2:
/build/reproducible-path/j4-dmenu-desktop-3.2/obj-x86_64-gnu/../src/main.cc:881:1: warning: ‘void do_wait_on(NotifyBase&, const char*, AppManager&, const stringlist_t&, RunPhase::CommandRetrievalLoop&, ExecutePhase::BaseExecutable*)’ defined but not used [-Wunused-function]
  881 | do_wait_on(NotifyBase &notify, const char *wait_on, AppManager &appm,
      | ^~~~~~~~~~
```

Also, I noticed that it is a bit difficult to know which notify implementation is used from the build logs, so adding this kind of patch would be nice:

```diff
--- a/src/meson.build
+++ b/src/meson.build
@@ -62,6 +62,7 @@ elif get_option('notify-implementation') == 'none'
   )
   notify = 'none'
 endif
+summary('notify implementation', notify)
 
 flags = ['-DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG']
 
```

_example_:
<img width="335" height="169" alt="2026-04-29-105005_335x169_scrot" src="https://github.com/user-attachments/assets/ef3a44f3-cb51-4485-b938-f609cdf6a5a7" />

--- a/src/meson.build
+++ b/src/meson.build
@@ -62,6 +62,7 @@ elif get_option('notify-implementation') == 'none'
   )
   notify = 'none'
 endif
+summary('notify implementation', notify)
 
 flags = ['-DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG']
 

